linux_os/guide/system/software/system-tools/package_libreport-plugin-rhtsupport_removed/rule.yml

```diff
@@ -19,7 +19,6 @@ identifiers:
 references:
     disa: CCI-000381
     srg: SRG-OS-000095-GPOS-00049
-    stigid@rhel8: RHEL-08-040001
```

Comment:
I decided to keep the rule as it is. It aligns with the spirit of the rule. In the GUI profile unfortunately we can't select it because it is Anaconda dependency. Do you agree with this?